test/e2e/pull_test.go

```diff
@@ -35,6 +35,24 @@ var _ = Describe("Podman pull", func() {
 
 	})
 
+	It("podman pull multiple images with tag", func() {
+		session := podmanTest.Podman([]string{"pull", "busybox:latest", "alpine:3"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
```

Comment:
We should also check the output string and make sure that the images are correctly reported as pulled.

Reply:
Look for `GrepString(...)` in the other tests.